### ISSUE TITLE
Exception upon file drop in IE11

### ISSUE THREAD

**Issue by alexandery** [danialfarid/ng-file-upload#1047]
Hi,

Thanks for the great directive! I have upgraded from one of the previous versions and am now getting an exception in IE11 when dropping a file onto a drop zone (this used to work just fine). I've tried both ng-file-uplad-all.js and ng-file-upload.js with the same result (naturally). This works in Chrome.

The exception:

> Unhandled exception at line 1981, column 7 in http://localhost:1322/file-upload/ng-file-upload-all.js
> 0x80070057 - JavaScript runtime error: Invalid argument.

The line of code:

```javascript
var html = (evt.dataTransfer && evt.dataTransfer.getData && evt.dataTransfer.getData('text/html'));
```

Here is my html:

```html
	<div class="drop-box"
		 ngf-drop
		 ngf-drag-over-class="{accept:'dragover', reject:'dragover-err', delay:100}"
		 ng-model="uploadHandler.files"
		 ngf-multiple="true"
		 ngf-allow-dir="false"
		 accept="image/*"
		 ngf-pattern="image/*">
		Attach images by dragging and dropping to this area
	</div>
	<div ngf-no-file-drop>
		File Drag/Drop is not supported for this browser
	</div>
```

And it never hits my javascript code.

**Comment by alexandery** [comment]
That was fast! Thank you! :thumbsup:

**Comment by danialfarid** [comment]
Fixed at version 9.0.7
